simplecpp.cpp

```diff
@@ -1020,15 +1020,15 @@ void simplecpp::TokenList::constFoldBitwise(Token *tok)
 {
     Token * const tok1 = tok;
     for (const char *op = "&^|"; *op; op++) {
-        const std::string* altop;
```

Comment:
I believe `altop` is a abbreviation for `alternative operator` or something like that.

Reply:
I will update this in approx. two weeks. I suppose "alternativeOp" could be used as local variable name. Ok?

Reply:
that is ok